intel_extension_for_transformers/llm/library/jblas/jblas/kernel_avx512f.h

```diff
@@ -959,7 +973,7 @@ inline void f32_f4_quantize_1x16(const float* srcptr, int8_t* dstptr, int ld_src
     if constexpr (F4_T == JBLAS_DTYPE::F4_BNB) sub_v = _mm512_set1_ps(F4_BNB_quant_sub_helper[i]);
     if constexpr (F4_T == JBLAS_DTYPE::F4_E2M1) sub_v = _mm512_set1_ps(F4_E2M1_quant_sub_helper[i]);
     zmm1 = _mm512_sub_ps(zmm0, sub_v);
-    mask0 = _mm512_cmple_ps_mask(zmm1, zmm_v0);
+    mask0 = _mm512_cmp_ps_mask(zmm1, zmm_v0, 2);
```

Comment:
what's the difference?

Reply:
we found `_mm512_cmple_ps_mask` can't pass compile on Red Hat default gcc-toolset (gcc version 8.3/9.3) with Debug flag.. 
the strange thing is that the compile error will disapper if with Relase flag.